packages/validator/templates/OAS2OIS.json

```diff
@@ -0,0 +1,288 @@
+{
+	"__actions": [
+		{
+			"__insert": {
+				"__target": "oisFormat",
+				"__value": "1.0.0"
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.servers",
+				"__value": []
+			}
+		},
+		{
+			"__insert": {
+				"__target": "endpoints",
+				"__value": []
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.security",
+				"__value": {}
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.components.securitySchemes",
+				"__value": {}
+			}
+		},
+		{
+			"__insert": {
+				"__target": "apiSpecifications.paths",
+				"__value": {}
+			}
+		}
+	],
+	"info": {
+		"title": {
+			"__actions": [
+				{
+					"__copy": {
+						"__target": "title"
+					}
+				}
+			]
+		},
+		"version": {
+			"__actions": [
+				{
+					"__copy": {
+						"__target": "version"
+					}
+				}
+			]
+		},
+		"__ignore": {}
+	},
+	"servers": {
+		"__arrayItem": {
+			"url": {
+				"__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$",
+				"__actions": [
+					{
+						"__copy": {
+							"__target": "apiSpecifications.servers[0].url"
+						}
+					}
+				]
+			}
+		}
+	},
+	"paths": {
+		"__keyRegexp": "^\\/[^\\s'\"\\\\]+$",
+		"__objectItem": {
+			"__keyRegexp": "^(get|post)$",
+			"__objectItem": {
+				"__actions": [
+					{
+						"__insert": {
+							"__target": "apiSpecifications.paths.{{1}}.{{2}}.parameters",
+							"__value": []
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[].name",
+							"__value": "{{1}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].operation.method",
+							"__value": "{{2}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].operation.path",
+							"__value": "{{1}}"
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].reservedParameters",
+							"__value": [
+								{
+									"name": "_type"
+								},
+								{
+									"name": "_path"
+								},
+								{
+									"name": "_times"
+								}
+							]
+						}
+					},
+					{
+						"__insert": {
+							"__target": "endpoints[_].fixedOperationParameters",
```

Comment:
Nit: similar to my previous comment. Is there a good reason for including `fixedOperationParameters` and not just leaving it out? Since OAS won't specify `fixedOperationParameters` since it's something that's defined on the oracle side.

Reply:
It's a required field so added as an empty array.